src/components/promotion/PromotionImage.tsx

```diff
@@ -19,6 +19,7 @@ export default function PromotionImage(props: Props): ReactElement {
   const defaultSrc =
     'https://d1ralsognjng37.cloudfront.net/92a7b4fb-892c-47bb-b5bb-884c89c254a2.jpeg';
 
+  // TODO: If getting the image from S3 fails, resort to a default image
```

Comment:
How difficult do you think this TODO would be, just curious because I think it would be beneficial. 

I think it may be as simple as this:
```
<div>{imageSrc ? <img style={styles.image} src={imageSrc} alt=""  onerror="if (this.src != 'error.jpg') this.src = 'error.jpg';" /> : <Skeleton.Image />}</div>
```

Reply:
Oh yes I'll add that!